src/tests/staking_reward_test.ts

```diff
@@ -35,7 +32,7 @@ describe("StakingReward", () => {
         date: "2024-05-01",
         amount: "361",
         state: StakingRewardStateEnum.Pending,
-        format: StakingRewardFormat.Usd,
+        format: "usd",
```

Comment:
Do we want to use the enum here?

Reply:
It would require me to use the StakingRewardFormat from the generated code (../client) which will cause a collision and require me to think of a better name. 

I was following what was done similarly for TransactionStatus like say [here](https://github.com/coinbase/coinbase-sdk-nodejs/blob/master/src/tests/trade_test.ts#L57) where we skip the problem by using naked strings. Note - this is only to be done at the time of setting up our testdata i.e. server response.

If you are keen on using the constant, I can do that as well. Not an issue.

Reply:
I am OK with either 👍. Just wanted to make sure it wasn't accidentally missed.